common/sheet_helpers.ts

```diff
@@ -114,7 +114,7 @@ export class SettingMap<P extends { [Property in keyof P]: P[keyof P] }>
     return this.keys.reduce(
       (prev, key) => {
         prev[key] =
-          (!(key in campaignValue) || campaignValue[key] === ''
+          (!(key in campaignValue) || !campaignValue[key]
```

Comment:
```suggestion
          (!(key in campaignValue && campaignValue[key])
```

What do you think @tabrethauer ? I know I wrote this line.

Reply:
sticking with this as I think it's DNF